server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
From: https://github.com/golang/go/search?utf8=%E2%9C%93&q=meta

// hasMeta reports whether path contains any of the magic characters
// recognized by Match.
func hasMeta(path string) bool {
	// TODO(niemeyer): Should other magic characters be added here?
	return strings.ContainsAny(path, "*?[")
}

232 func TestMatchGoImport(t *testing.T) {
233		tests := []struct {
234			imports []metaImport
235			path    string
236			mi      metaImport
237			err     error
238		}{...

https://golang.org/pkg/regexp/#QuoteMeta

func QuoteMeta(s string) string
QuoteMeta returns a string that quotes all regular expression metacharacters inside the argument text; the returned string is a regular expression matching the literal text. For example, QuoteMeta(`[foo]`) returns `\[foo\]`.

Now, while you all are looking up what "meta" means in a dictionary, I am a native speaker of English, and a former Google SRE who fixed some code in Google's internal Go RPC library.

Since "md" is already taken up by a map of MethodDescs, "meta" is the appropriate name for the unexported variable here.